my_game.py

```diff
@@ -249,7 +251,7 @@ class IntroView(arcade.View):
     View to show instructions
     """
 
-    def on_show_view(self):
+    def on_show_view(self, fullscreen=True):
```

Comment:
Please remove this change. It's not related to fonts. No decision has been made to do this.